app/src/main/java/org/stacks/app/data/EncryptedPreferencesIdentityRepository.kt

```diff
@@ -34,7 +35,6 @@ class EncryptedPreferencesIdentityRepository
     override fun observe(): Flow<List<IdentityModel>> =
         { preferences }
             .toFlow()
-            .filter { !PolluteString.isPolluted(it) }
```

Comment:
But wasn't this needed? To tell if the value is empty? Or the `toListIdentities` already covers it?